kubernetes/samples/scripts/domain-lifecycle/stopCluster.sh

```diff
@@ -85,22 +85,30 @@ initialize() {
 
 initialize
 
-# Get the domain in json format
+# Get the domain in json format. Changed made on 08/16/2022
 domainJson=$(${kubernetesCli} get domain ${domainUid} -n ${domainNamespace} -o json --ignore-not-found)
+clusterJson=$(${kubernetesCli} get cluster ${clusterName} -n ${domainNamespace} -o json --ignore-not-found)
+printInfo "clusterJson content before changes: ${clusterJson}"
 if [ -z "${domainJson}" ]; then
   printError "Unable to get domain resource for domain '${domainUid}' in namespace '${domainNamespace}'. Please make sure the 'domain_uid' and 'namespace' specified by the '-d' and '-n' arguments are correct. Exiting."
   exit 1
 fi
 
+if [ -z "${clusterJson}" ]; then
+  printError "Unable to get cluster resource for cluster '${clusterName}' in namespace '${domainNamespace}'. Please make sure the 'clusterName and 'namespace' specified by the '-d' and '-n' arguments are correct. Exiting."
```

Comment:
The script is assuming that cluster resource will always exist and it's going to fail if the cluster resource doesn't exist. My understanding is that cluster resource may not exist for the default behavior. @rjeberhard The expected behavior of when the cluster resource doesn't exist needs to be decided.

Reply:
I just discussed this with QA folks in today's weekly team meeting, we feel that we should keep the original logic in the scripts - fail the test if there is no cluster exists. The reason is that for this test suite, we are testing "cluster" related operations, like restart, rollingrestart, scale, etc. an existing cluster is a must for the tests to pass and test suite is responsible to create it and then let the script now. If we let the script to create the cluster if it's missing for any reason, it may confuse the test suite and make a mess later.

Reply:
Changed error msg to "Unable to get cluster resource for cluster '${clusterName}' in namespace '${domainNamespace}'. Please make sure that a Cluster exists for cluster '${clusterName}' and that this Cluster is referenced by the Domain."